### ISSUE TITLE
Gem library loading

### ISSUE THREAD

**Issue by 60-hz** [plugdata-team/plugdata#1450]
In plugdata, Gem is not added in startup lib by default.
User need a [declare -lib Gem] to load Gem,  and no lib informations is printed (gem version etc...)

**Comment by timothyschoen** [comment]
Yeah, this is on purpose. The problem is that ELSE also has a [circle] object, cyclone also has a [scale] object, etc. 

If you want to have Gem in global scope always, you can go into Settings -> Paths and add Gem to the libraries list.

**Comment by 60-hz** [comment]
Ok, I see. But I think name clash should be avoided inside PlugData because that's a source of confusion for users.
I already opened an issue about name clash for [circle] in else.
For cyclone... since else can do all that cyclone do... can't it be optional?
